src/app/state/account/selectors.ts

```diff
@@ -11,3 +11,4 @@ export const selectTransactionsError = createSelector([selectAccount], account =
 export const selectAccountAddress = createSelector([selectAccount], account => account.address)
 export const selectAccountAvailableBalance = createSelector([selectAccount], account => account.available)
 export const selectAccountIsLoading = createSelector([selectAccount], account => account.loading)
+export const selectAccountAllowances = createSelector([selectAccount], account => account.allowances || [])
```

Comment:
Looks like paratimes are using wrong account information

Reproduce steps
- create a wallet and open paratimes: http://localhost:3000/account/oasis1qztn43zuxtjyk49q6ee3fmy3cjyn6lk0fscnrsnh/paratimes
- click home and create a new wallet: http://localhost:3000/account/oasis1qq0xldx5xu2a05gdhn7l3h9w4u44ucx53qw7cex4
- click back until http://localhost:3000/account/oasis1qztn43zuxtjyk49q6ee3fmy3cjyn6lk0fscnrsnh/paratimes
- now the opened account is oasis1qq0xldx5xu2a05gdhn7l3h9w4u44ucx53qw7cex4 ("wallet" state)
- but viewed account is oasis1qztn43zuxtjyk49q6ee3fmy3cjyn6lk0fscnrsnh ("account" state)

(or open multiple accounts, open paratimes on one, select another account, and click back)

Issues:
- paratimes flow uses balance and other info from viewed account
- paratimes form should be hidden on foreign accounts, see
  https://github.com/oasisprotocol/oasis-wallet-web/pull/865 and https://github.com/oasisprotocol/oasis-wallet-web/pull/918

Reply:
as long as I understand the problem correctly https://github.com/oasisprotocol/oasis-wallet-web/pull/978 covers this with a redirect. Basically, user won't be even able to open  "foreign" /paratime route, because there is no value in such action.